orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -285,19 +285,19 @@ def save_as_onnx(self, path):
         with open(path, "wb") as f:
             f.write(self._onnx_model.SerializeToString())
 
-    def _debug_model_export(self, input):
+    def _check_model_export(self, input):
```

Comment:
Should it be "check" or "verify"?

Reply:
i don't have a strong preference over the two. the update is to have it consistent with the flag.